src/hashbang.sh

```diff
@@ -358,6 +358,7 @@ if [ -n "$public_key" ] && [ -n "$username" ]; then
 			printf '\nHost hashbang\n  HostName %s\n  IdentitiesOnly yes\n  User %s\n  IdentityFile %s\n' \
 			       "${host}" "$username" "$private_keyfile" \
 			>> ~/.ssh/config
+			chmod 600 ~/.ssh/config
```

Comment:
maybe `-g,-o` instead? or is that not posix?

(i.e. if user had umask to cause 400, we don't want to un-do it)

Reply:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/chmod.html looks like it is POSIX